src/main/java/me/zeroeightsix/kami/module/modules/render/MapPeview.kt

```diff
@@ -0,0 +1,41 @@
+package me.zeroeightsix.kami.module.modules.render
+
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.EventHook
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Settings
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.item.ItemMap
+import net.minecraft.world.World
+import net.minecraftforge.client.event.RenderTooltipEvent
+
+
+/**
+ * Created by fred41 on 27/05/2020.
+ *
+ */
+@Module.Info(
+        name = "MapPreview",
+        category = Module.Category.RENDER,
+        description = "Shows the contents of a map when you hover over one"
+)
+class MapPeview : Module() {
+    private val scale = register(Settings.doubleBuilder("scale").withRange(0.0, 3.0).withValue(0.5).build())
```

Comment:
your value can't be bigger then your range, did you mean to make the range 0.0 to 0.5?

Reply:
it thought this ment with a range form 0 to 3  with default value 0.5

Reply:
oh sorry, I misread that.